### DISCUSSION TITLE
Amazon Ami suggestion for yolov8

### DISCUSSION THREAD

**Discussion by hilaloztemelbottobo** [ultralytics/ultralytics#6507] [Q&A]
Do you have a specific suggestion for an Amazon AMI on AWS Deep Learning Instance for yoloV8?

**Comment by pderrenger** [comment]
@HilalOztemelBottobo hello! For running YOLOv8 on an AWS Deep Learning AMI, I would recommend choosing an AMI that comes with the latest supported versions of PyTorch and CUDA, as YOLOv8 is optimized for PyTorch. Make sure the instance type you select has a suitable GPU for your workload, such as the `p3` or `g4dn` series, which offer NVIDIA V100 and T4 GPUs respectively.

Once you've set up your instance, you can install YOLOv8 by following the installation instructions in our documentation. If you encounter any issues or have further questions, feel free to reach out on the GitHub issues page for assistance. Happy training! 😊🚀
